src/constellation/config.py

```diff
@@ -67,6 +67,10 @@ def config_vault(data, path):
     return vault.VaultConfig(url, auth_method, auth_args)
 
 
+def config_acme(data):
```

Comment:
Is it worth passing in a path in here, so that we don't have to make assumptions about the top-level structure of the data? Similar to how `config_vault` does it.